src/client/components/Navbar.jsx

```diff
@@ -65,8 +65,8 @@ const Navbar = () => {
                       </Link>
                     </li>
                     <li className="nav-item">
-                      <Link href="/work/create">
-                        <a className="nav-link">Create Work</a>
+                      <Link href="/profile/profilelist">
```

Comment:
What about `/profile/list` instead of `/profile/profilelist` because it removes bit of redundancy.

Reply:
Good idea! Fixed!